src/parser/jobs/sam/modules/Sen.tsx

```diff
@@ -52,15 +53,9 @@ export default class Sen extends Module {
 
 		// Death, as well as all Iaijutsu, remove all available sen
 		this.addHook('cast', {by: 'player', abilityId: IAIJUTSU}, this.remove)
+		this.addHook('cast', {by: 'player', abilityId: TSUBAME}, this.overwrite)
```

Comment:
OOI, why does using a Tsubame skill discount a previous sen waste?

Reply:
Mike filled me in on the reasoning. To avoid cutting down mistakes that were actual mistakes, probably worth tracking yukikaze overcaps seperately, and only removing one yuki overcap per tsubame.